src/jdk.hotspot.agent/doc/transported_core.html

```diff
@@ -10,101 +10,87 @@ <h1>Debugging transported core dumps</h1>
+always successfully open the core dump. This is due to  library (shared
+objects or DLLs) mismatches between the core dump machine and the debugger machine.
+For this reason you should first consider using one of the following approaches to
+debugging the core dump rather than transporting it to a different machine:
+<ul>
+  <li>Remote login to the machine that the core dump was produced on,
+    and use <a href="clhsdb.html">CLHSDB - SA command line HSDB interface</a>.
+  <li>Use SA remote debugging support to remotely debug the core directly on the machine
+   it was produced on. This is done by first running <b>jhsdb debugd</b> on the machine with the core dump,
+    and then attaching to it from another machine by using the jhsdb <b>--connect</b> argument. See
+    the <b>jhsdb</b> man page for details.
+</ul>
 </p>
 
 <p>
-In most platforms, core dumps do not contain text (a.k.a) Code pages.
-There pages are to be read from executable and shared objects (or DLLs).
-Therefore it is important to have matching executable and shared object
-files in debugger machine. 
+With transported core dumps, SA may produce an error message or throw
+an exception (such as for a missing hotspot symbol) when opening the core dump.
+Environment variable <b>LIBSAPROC_DEBUG</b> may be set to any value
+to help debug the root casue of these failures. With <b>LIBSAPROC_DEBUG</b> set, SA prints many
+messages in standard error which can be useful for further debugging. Note that
+<b>LIBSAPROC_DEBUG</b> is not supported on Windows.
 </p>
 
-<h3>Solaris transported core dumps</h3>
-
 <p>
-Debuggers on Solaris (and Linux) use two additional shared objects
-<b>rtld_db.so</b> and <b>libthread_db.so</b>. rtld_db.so is used to
-read information on shared objects from the core dump. libthread_db.so
-is used to get information on threads from the core dump. rtld_db.so
-evolves along with rtld.so (the runtime linker library) and libthread_db.so
-evolves along with libthread.so (user land multithreading library). 
-Hence, debugger machine should have right version of rtld_db.so and
-libthread_db.so to open the core dump successfully. More details on
-these debugger libraries can be found in 
-<a href="http://docs.sun.com/app/docs/doc/817-1984/">
-Solaris Linkers and Libraries Guide - 817-1984</a>
+On  most platforms, core dumps do not contain text (code) pages.
+Their pages are instead  read from the executable and shared objects (or DLLs).
+Therefore it is important to have a matching java executable and shared object
+files on the debugger machine. The best  way to guarantee this match is to match the
+debugger machine to that of core dump machine. This means having the same
+OS version and libraries, and also having the same version of the JDK. It also means having
+the OS libraries and JDK installed in the same locations on both machines. However, often this isn't an option,
+and instead you need to include copies of the core dump machine's libraries
```

Comment:
Perhaps, it can be rephrased as follows:

... instead you need to bring copies of the libraries and java installation from the machine where the core dump was produced to the debugger machine, and ...